src/components/expanded-state/UniqueTokenExpandedState.tsx

```diff
@@ -261,6 +273,13 @@ const UniqueTokenExpandedState = ({
   const ensProfile = useENSProfile(cleanENSName, { enabled: isENS });
   const ensData = ensProfile.data;
 
+  const profileInfoSectionAvailable = useMemo(() => {
+    const available = Object.keys(ensData?.records || {}).some(
+      key => key !== ENS_RECORDS.cover && key !== ENS_RECORDS.avatar
```

Comment:
don't we want to show the profile info section if we have a cover photo? the cover photo is apart of it too

Reply:
oh right